session.go

```diff
@@ -71,6 +71,7 @@ type Session interface {
 	DropPreparedStmt(stmtID uint32) error
 	SetClientCapability(uint32) // Set client capability flags.
 	SetConnectionID(uint64)
+	SetCharset(cs, co string)
```

Comment:
Why not define `SetCollation` directly with a single integer argument?

Reply:
good idea.